Vagrantfile

```diff
@@ -1,11 +1,19 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Automated creation of testing environments on misc. platforms
+# Automated creation of testing environment and standalones of borg-backup for various platforms.
```

Comment:
It's fine, just fix the few places. I am also new to vagrant (and I like it). :)